fuzz/meson.build

```diff
@@ -9,11 +9,11 @@ libass_fuzz = executable(
     install: false,
     include_directories: incs,
     dependencies: deps,
-    link_with: libass_for_tools,
+    objects: libass.extract_all_objects(recursive: true),
```

Comment:
Here too `libass_for_tools` should be kept

Reply:
Updated.